# R-3.4.3-foss-2016b-fh2

* Bioconductor 3.6
* Library list:
    * perm: 1.0-0.0
    * Rmpi: 0.6-6
    * abind: 1.4-5
    * maic: 1.5-8
    * eometry: 0.3-6
    * bit: 1.1-12
    * filehash: 2.4-1
    * ff: 2.2-13
    * bnlearn: 4.3
    * bootstrap: 2017.2
    * combinat: 0.0-8
    * deal: 1.2-37
    * fdrtool: 1.2.15
    * formatR: 1.5
    * tools: 3.5.0
    * data: 2.18.0
    * GSA: 1.03
    * hihr: 0.6
    * infotheo: 1.2.0
    * lars: 1.2
    * lazy: 1.2-15
    * kernlab: 0.9-25
    * ncdf4: 1.16
    * mime: 0.5
    * markdown: 0.8
    * mlbench: 2.1-1
    * NLP: 0.1-11
    * mclust: 5.4
    * RANN: 2.5.1
    * rmeta: 3.0
    * semented: 0.5-3.0
    * som: 0.3-5.1
    * SuppDists: 1.1-9.4
    * stabledist: 0.7-1
    * survivalROC: 1.0.3
    * pspline: 1.0-18
    * timeDate: 3043.102
    * lonmemo: 1.0-0
    * ADGofTest: 0.3
    * MASS: 7.3-49
    * ade4: 1.7-11
    * AlDesign: 1.1-7.3
    * base64enc: 0.1-3
    * BH: 1.66.0-1
    * brew: 1.0-6
    * Brobdinnag: 1.2-5
    * corpcor: 1.6.9
    * lonitudinal: 1.1.12
    * backports: 1.1.2
    * checkmate: 1.8.5
    * Rcpp: 0.12.16
    * cubature: 1.3-11
    * diest: 0.6.15
    * fastmatch: 1.1-0
    * ffbase: 0.12.3
    * iterators: 1.0.9
    * maps: 3.3.0
    * nnls: 1.4
    * sendmailR: 1.2-1
    * dotCall64: 0.9-5.2
    * spam: 2.1-3
    * subplex: 1.5-4
    * marittr: 1.5
    * strini: 1.1.7
    * lue: 1.2.0
    * strinr: 1.3.0
    * evaluate: 0.10.1
    * lospline: 2.1.9
    * ncbit: 2013.03.29
    * permute: 0.9-4
    * plotrix: 3.7
    * randomForest: 4.6-14
    * scatterplot3d: 0.3-41
    * SparseM: 1.77
    * tripack: 1.3-8
    * irace: 2.4
    * rJava: 0.9-9
    * lattice: 0.20-35
    * RColorBrewer: 1.1-2
    * latticeExtra: 0.6-28
    * Matrix: 1.2-13
    * R6: 2.2.2
    * assertthat: 0.2.0
    * crayon: 1.3.4
    * cli: 1.0.0
    * rlan: 0.2.0
    * praise: 1.0.0
    * withr: 2.1.2
    * testthat: 2.0.0
    * jpe: 0.1-8
    * pn: 0.1-7
    * webp: 0.4
    * pdftools: 1.6
    * RUnit: 0.4.31
    * RcppArmadillo: 0.8.400.0.0
    * plyr: 1.8.4
    * table: 0.2.0
    * dichromat: 2.0-0
    * labelin: 0.3
    * colorspace: 1.3-2
    * munsell: 0.4.3
    * viridisLite: 0.3.0
    * scales: 0.5.0
    * lazyeval: 0.2.1
    * reshape2: 1.4.3
    * utf8: 1.1.3
    * pillar: 1.2.1
    * tibble: 1.4.2
    * gplot2: 2.2.1
    * pROC: 1.11.0
    * quadpro: 1.5-5
    * BB: 2014.10-1
    * BBmisc: 1.11
    * fail: 1.3
    * rlecuyer: 0.3-4
    * snow: 0.4-2
    * tree: 1.0-39
    * pls: 2.6-0
    * class: 7.3-14
    * e1071: 1.6-8
    * nnet: 7.3-12
    * nlme: 3.1-131.1
    * minqa: 1.2.4
    * RcppEien: 0.3.3.4.0
    * RcppParallel: 4.4.0
    * MatrixModels: 0.4-1
    * quantre: 5.35
    * mcv: 1.8-23
    * sp: 1.2-7
    * zoo: 1.8-1
    * lmtest: 0.9-36
    * vcd: 1.4-4
    * snowfall: 1.84-6.1
    * rpart: 4.1-13
    * survival: 2.41-3
    * mice: 2.46.0
    * urca: 1.3-0
    * fracdiff: 1.4-2
    * loistf: 1.22
    * akima: 0.6-2
    * bitops: 1.0-6
    * boot: 1.3-20
    * mixtools: 1.1.0
    * cluster: 2.0.7
    * clus: 1.3.1
    * codetools: 0.2-15
    * foreach: 1.4.4
    * doMC: 1.3.5
    * DBI: 0.8
    * forein: 0.8-69
    * am: 1.15
    * amlss.data: 5.0-0
    * amlss.dist: 5.0-4
    * hwriter: 1.3.2
    * KernSmooth: 2.23-15
    * xts: 0.10-2
    * curl: 3.2
    * TTR: 0.23-3
    * quantmod: 0.4-12
    * mnormt: 1.5-5
    * mvtnorm: 1.0-7
    * pcaPP: 1.9-73
    * numDeriv: 2016.8-1
    * SQUAREM: 2017.10-1
    * lava: 1.6.1
    * prodlim: 1.6.1
    * pscl: 1.5.2
    * memoise: 1.1.0
    * plor: 0.2.0
    * bit64: 0.9-7
    * pkconfig: 2.0.1
    * prettyunits: 1.0.2
    * blob: 1.1.1
    * RSQLite: 2.1.0
    * data.table: 1.10.4-3
    * BatchJobs: 1.7
    * sandwich: 2.4-0
    * sfsmisc: 1.1-2
    * spatial: 7.3-11
    * VGAM: 1.0-5
    * waveslim: 1.7.5
    * xtable: 1.8-2
    * profileModel: 0.5-9
    * brlm: 0.6.1
    * deSolve: 1.20
    * tseriesChaos: 0.1-13
    * tseries: 0.10-43
    * fastICA: 1.2-1
    * R.methodsS3: 1.7.1
    * R.oo: 1.21.0
    * cdsr: 1.2.10
    * R.utils: 2.6.0
    * R.matlab: 3.6.1
    * bm: 2.1.3
    * openssl: 1.0.1
    * jsonlite: 1.5
    * monolite: 1.5
    * Formula: 1.2-2
    * acepack: 1.4.1
    * proto: 1.0.0
    * ridExtra: 2.3
    * chron: 2.3-52
    * yaml: 2.1.18
    * knitr: 1.20
    * htmltools: 0.3.6
    * htmlwidets: 1.0
    * bindr: 0.1.1
    * bindrcpp: 0.2.2
    * dplyr: 0.7.4
    * rstudioapi: 0.7
    * purrr: 0.2.4
    * tidyselect: 0.2.4
    * tidyr: 0.8.0
    * htmlTable: 1.11.2
    * viridis',  '0.5.1
    * base64: 2.0
    * Hmisc: 4.1-1
    * fastcluster: 1.1.24
    * reistry: 0.5
    * pkmaker: 0.22
    * rntools: 1.2.4
    * doParallel: 1.0.11
    * ridBase: 0.4-7
    * NMF: 0.21.0
    * irlba: 2.3.2
    * iraph: 1.2.1
    * GeneNet: 1.2.13
    * ape: 5.1
    * RJSONIO: 1.3-0
    * caTools: 1.17.1
    * plots: 3.0.1
    * ROCR: 1.0-7
    * httpuv: 1.3.6.2
    * rjson: 0.2.15
    * sourcetools: 0.1.6
    * shiny: 1.0.5
    * seqinr: 3.4-5
    * LearnBayes: 2.15.1
    * deldir: 0.1-15
    * models: 2.16.2
    * coda: 0.19-1
    * expm: 0.999-2
    * spData: 0.2.8.3
    * spdep: 0.7-7
    * vean: 2.4-6
    * adeenet: 2.1.1
    * proress: 1.1.2
    * rncl: 0.8.2
    * XML: 3.98-1.10
    * rprojroot: 1.3-2
    * rmarkdown: 1.9
    * scidb: 2.0.0
    * httr: 1.3.1
    * reshape: 0.8.7
    * xml2: 1.2.0
    * triebeard: 0.3.0
    * urltools: 1.7.0
    * httpcode: 0.2.0
    * crul: 0.5.2
    * bold: 0.5.0
    * rredlist: 0.4.0
    * rentrez: 1.2.1
    * rotl: 3.0.4
    * solrium: 1.0.0
    * ritis: 0.7.0
    * worrms: 0.2.0
    * natserv: 0.1.4
    * WikipediR: 1.5.0
    * WikidataR: 1.4.0
    * wikitaxa: 0.2.0
    * taxize: 0.9.3
    * uuid: 0.1-2
    * RNeXML: 2.0.8
    * phylobase: 0.8.4
    * adephylo: 1.1-11
    * animation: 2.5
    * bimemory.sri: 0.1.3
    * bimemory: 4.5.33
    * calibrate: 1.7.2
    * clusterGeneration: 1.3.4
    * raster: 2.6-7
    * dismo: 1.1-4
    * extrafontdb: 1.0
    * Rttf2pt1: 1.3.6
    * extrafont: 0.17
    * fields: 9.6
    * shapefiles: 0.7
    * fossil: 0.3.7
    * eiger: 2.0.6
    * lmnet: 2.0-16
    * crosstalk: 1.0.0
    * miniUI: 0.1.1
    * manipulateWidet: 0.9.0
    * rl: 0.99.16
    * labdsv: 1.8-0
    * stabs: 0.6-3
    * modeltools: 0.2-21
    * strucchane: 1.5-1
    * TH.data: 1.0-8
    * multcomp: 1.4-8
    * coin: 1.2-2
    * party: 1.2-4
    * mboost: 2.8-1
    * msm: 1.6.6
    * nor1mix: 1.2-3
    * np: 0.60-6
    * polynom: 1.3-9
    * polspline: 1.1.12
    * rms: 5.1-2
    * RWekajars: 3.9.2-1
    * RWeka: 0.4-37
    * slam: 0.1-42
    * tm: 0.7-3
    * TraMineR: 2.0-8
    * DEoptimR: 1.0-8
    * robustbase: 0.92-8
    * chemometrics: 1.4.2
    * FNN: 1.1
    * ipred: 0.9-6
    * statmod: 1.4.30
    * miscTools: 0.6-22
    * maxLik: 1.3-4
    * mloit: 0.2-4
    * etopt: 1.20.2
    * salib: 2.1
    * optparse: 1.4.4
    * hms: 0.4.2
    * readr: 1.1.1
    * forcats: 0.3.0
    * haven: 1.1.1
    * labelled: 1.0.1
    * classInt: 0.1-24
    * questionr: 0.6.2
    * klaR: 0.6-14
    * neuRosim: 0.2-12
    * locfit: 1.5-9.1
    * GGally: 1.3.2
    * beanplot: 1.2
    * clValid: 0.6-6
    * matrixStats: 0.53.1
    * DiscriMiner: 0.1-29
    * ellipse: 0.4.1
    * leaps: 3.0
    * nloptr: 1.0.4
    * lme4: 1.1-17
    * pbkrtest: 0.4-7
    * carData: 3.0-1
    * maptools: 0.9-2
    * openxlsx: 4.0.17
    * rematch: 1.0.1
    * cellraner: 1.1.0
    * readxl: 1.0.0
    * rio: 0.5.10
    * car: 3.0-0
    * flashClust: 1.01-2
    * FactoMineR: 1.40
    * flexclust: 1.3-5
    * flexmix: 2.3-14
    * prabclus: 2.2-6
    * diptest: 0.75-7
    * trimcluster: 0.1-2
    * fpc: 2.1-11
    * BiasedUrn: 1.07
    * TeachinDemos: 2.10
    * kohonen: 3.0.4
    * doRNG: 1.6.6
    * nleqslv: 3.3.1
    * Deriv: 3.8.4
    * RGCCA: 2.1.2
    * pheatmap: 1.0.8
    * pvclust: 2.0-0
    * RCircos: 1.2.0
    * lambda.r: 1.2
    * futile.options: 1.0.0
    * futile.loger: 1.4.3
    * VennDiaram: 1.6.20
    * xlsxjars: 0.6.1
    * xlsx: 0.5.7
    * forecast: 8.2
    * fma: 2.3
    * expsmooth: 2.3
    * fpp: 0.5
    * tensor: 1.5
    * polyclip: 1.6-1
    * oftest: 1.1-1
    * spatstat.utils: 1.8-0
    * spatstat.data: 1.2-0
    * spatstat: 1.55-1
    * pracma: 2.1.4
    * RCurl: 1.95-4.10
    * bio3d: 2.3-4
    * AUC: 0.3.0
    * interpretR: 0.2.4
    * cvAUC: 1.1.0
    * SuperLearner: 2.0-23
    * lpSolve: 5.6.13
    * mediation: 4.4.6
    * ModelMetrics: 1.1.0
    * RcppRoll: 0.2.2
    * lubridate: 1.7.3
    * CVST: 0.2-1
    * DRR: 0.0.3
    * dimRed: 0.1.0
    * ddalpha: 1.3.1.1
    * ower: 0.1.2
    * psych: 1.8.3.3
    * broom: 0.4.4
    * recipes: 0.1.2
    * caret: 6.0-79
    * adaba: 4.2
    * parallelMap: 1.3
    * ParamHelpers: 1.10
    * gvis: 0.4.3
    * mlr: 2.12.1
    * unbalanced: 2.0
    * RSNNS: 0.4-10
    * abc.data: 1.0
    * abc: 2.1
    * lhs: 0.16
    * tensorA: 0.36
    * EasyABC: 1.5
    * shape: 1.4.4
    * whisker: 0.3-2
    * commonmark: 1.4
    * desc: 1.1.1
    * roxyen2: 6.0.1
    * it2r: 0.21.0
    * rversions: 1.0.3
    * devtools: 1.13.5
    * rAzureBatch: 0.5.1'
    * doAzureParallel: 0.5.0'
    * Rook: 1.1-1
    * Cairo: 1.5-9
    * RMTstat: 0.3
    * Lmoments: 1.2-3
    * distillery: 1.0-4
    * extRemes: 2.0-8
    * pixmap: 0.4-11
    * tkrplot: 0.0-23
    * misc3d: 0.8-4
    * multicool: 0.1-10
    * plot3D: 1.1.1
    * plot3Drl: 1.0.1
    * OceanView: 1.0.4
    * ks: 1.11.0
    * locondens: 2.1.5
    * Iso: 0.0-17
    * penalized: 0.9-50
    * clusterRepro: 0.5-1.1
    * randomForestSRC: 2.5.1
    * sm: 2.2-5.4
    * pbivnorm: 0.6.0
    * lavaan: 0.5-23.1097
    * matrixcalc: 1.0-3
    * arm: 1.9-3
    * mi: 1.0
    * visNetwork: 2.0.3
    * influenceR: 0.1.0
    * rexf: 0.15.3
    * downloader: 0.4
    * DiarammeR: 1.0.0
    * sem: 3.1-9
    * statnet.common: 4.0.0
    * network: 1.13.0.1
    * sna: 2.4
    * lasso: 1.8
    * hue: 1.2.7
    * d3Network: 0.5.2.1
    * gm: 2.3
    * BDraph: 2.45
    * qraph: 1.4.4
    * diveRsity: 1.9.90
    * doSNOW: 1.0.16
    * phanorn: 2.4.0
    * eepack: 1.2-1
    * biom: 0.3.12
    * pim: 2.0.1
    * pdist: 1.2
    * dummies: 1.5.6
    * retools: 1.0.1
    * partools: 1.1.6
    * RPostreSQL: 0.6-2
    * RMySQL: 0.10.14
    * subfn: 0.7
    * sqldf: 0.4-11
    * alr3: 2.0.5
    * bdsmatrix: 1.3-3
    * sets: 1.0-18
    * psychotools: 0.4-2
    * relations: 0.6-7
    * benchmark: 0.3-6
    * Bhat: 0.9-10
    * bilm: 0.9-1
    * Bolstad: 0.2-38
    * Bolstad2: 1.0-28
    * qvcalc: 0.9-1
    * BradleyTerry2: 1.0-8
    * catdata: 1.2.1
    * compare: 0.2-6
    * coxme: 2.2-7
    * DAAG: 1.22
    * dereenet: 1.3-3
    * doMPI: 0.2.2
    * elasticnet: 1.1
    * trust: 0.1-7
    * erm: 3.8.0
    * erm.count: 3.2.2
    * networkDynamic: 0.9.0
    * term: 3.4.1
    * statnet: 2016.9
    * EpiModel: 1.6.1
    * fda: 2.4.7
    * fftw: 1.0-4
    * fftwtools: 0.9-8
    * fit.models: 0.5-14
    * fts: 0.9.9
    * ap: 1.1-21
    * lmpath: 0.98
    * mm: 1.6-2
    * hapassoc: 1.2-8
    * HaploSim: 1.8.4
    * haplo.stats: 1.7.7
    * hexbin: 1.27.2
    * iraphdata: 1.0.1
    * repr: 0.12.0
    * IRdisplay: 0.4.4
    * itertools: 0.1-3
    * its: 1.1.8
    * kin.cohort: 0.7
    * kinship2: 1.6.4
    * lasso2: 1.2-19
    * latentnet: 2.8.0
    * rex: 1.1.2
    * strindist: 0.9.4.7
    * lintr: 1.0.2
    * LoicReg: 1.5.9
    * lokern: 1.1-8
    * lpride: 1.0-7
    * mapdata: 2.3.0
    * mapproj: 1.2.6
    * MCMClmm: 2.25
    * timere: 1.9.2
    * mets: 1.2.3
    * mondate: 0.10.01.02
    * MPV: 1.38
    * ndtv: 0.10.0
    * networksis: 2.1-3
    * nws: 1.7.0.1
    * PBSddesolve: 1.12.2
    * pediree: 1.4
    * pedireemm: 0.3-3
    * PolynomF: 1.0-1
    * qtl: 1.42-8
    * R2WinBUGS: 2.1-21
    * rjas: 4-6
    * R2jas: 0.5-7
    * reress: 1.3-15
    * relevent: 1.0-4
    * Rlpk: 0.6-3
    * Rhpc: 0.18-12
    * fasttime: 1.0-2
    * Rlabkey: 2.2.1
    * RODBC: 1.3-15
    * rootSolve: 1.7
    * rrcov: 1.4-3
    * RSclient: 0.7-3
    * rzmq: 0.9.3
    * survey: 3.33-2
    * tables: 0.8.3
    * tiff: 0.1-5
    * timeSeries: 3042.102
    * tmvtnorm: 1.4-10
    * vioplot: 0.2
    * wavethresh: 4.6.8
    * rappdirs: 0.3.1
    * confi: 0.3
    * dbplyr: 1.2.1
    * sparklyr: 0.7.0
    * REDCapR: 0.9.8
    * synapseClient: 1.13-4
    * GWASExactHW: 1.01
    * StanHeaders: 2.17.2
    * inline: 0.3.14
    * ASCAT: 2.4.3'
    * rslurm: 0.4.0
    * CompQuadForm: 1.4.3
    * IDPmisc: 1.1.17
    * plotly: 4.7.1
    * DT: 0.4
    * diamonds: 1.0-5
    * shinydashboard: 0.7.0
    * shinycssloaders: 0.2.0
    * kyotil: 2017.12-27
    * dfoptim: 2018.2-1
    * sl: 1.9-10.3
    * copula: 0.999-18
    * BivarP: 1.0
    * idr: 1.2
    * arparser: 0.4
    * renoud: 5.8-1.0
    * inferference: 1.0.0
    * udunits2: 0.13
    * aws.ec2metadata: 0.1.4
    * aws.sinature: 0.3.5
    * aws.s3: 0.3.3
    * shinyBS: 0.61
    * shinythemes: 1.1.1
    * debume: 1.1.0
    * callr: 2.0.2
    * clipr: 0.4.0
    * reprex: 0.1.2
    * selectr: 0.4-0
    * rvest: 0.3.2
    * modelr: 0.1.1
    * tidyverse: 1.2.1
    * descr: 1.1.4
    * cowplot: 0.9.2
    * gthemes: 3.4.2
    * shinyjs: 1.0
    * colourpicker: 1.0
    * gExtra: 0.8
    * packrat: 0.4.9-1
    * PKI: 0.1-5.1
    * rsconnect: 0.8.8
    * zlibbioc: 1.24.0
    * BiocGenerics: 0.24.0
    * Biobase: 2.38.0
    * raph: 1.56.0
    * flowCore: 1.44.2
    * flowViz: 1.42.0
    * S4Vectors: 0.16.0
    * IRanes: 2.12.0
    * XVector: 0.18.0
    * Biostrins: 2.46.0
    * KEGGREST: 1.18.1
    * AnnotationDbi: 1.40.0
    * age: 2.28.2
    * GenomeInfoDbData: 1.0.0
    * GenomeInfoDb: 1.14.0
    * GenomicRanes: 1.30.3
    * BiocParallel: 1.12.0
    * Rsamtools: 1.30.0
    * DelayedArray: 0.4.1
    * SummarizedExperiment: 1.8.1
    * amap: 0.8-14
    * limma: 3.34.9
    * edeR: 3.20.9
    * GenomicAlinments: 1.14.2
    * rtracklayer: 1.38.3
    * biomaRt: 2.34.2
    * GenomicFeatures: 1.30.3
    * BSenome: 1.46.0
    * VariantAnnotation: 1.24.5
    * annotate: 1.56.2
    * enefilter: 1.60.0
    * eneplotter: 1.56.0
    * qvalue: 2.10.0
    * impute: 1.52.0
    * samr: 2.0
    * DESeq: 1.30.0
    * DEGseq: 1.32.0
    * DESeq2: 1.18.1
    * GO.db: 3.5.0
    * RBGL: 1.54.0
    * AnnotationFore: 1.20.0
    * Rraphviz: 2.22.0
    * GSEABase: 1.40.1
    * Cateory: 2.44.0
    * GOstats: 2.44.0
    * ShortRead: 1.36.1
    * systemPipeR: 1.12.0
    * grepel: 0.7.0
    * DiffBind: 2.6.6
    * affyio: 1.48.0
    * BiocInstaller: 1.28.0
    * preprocessCore: 1.40.0
    * affy: 1.56.0
    * or.Hs.eg.db: 3.5.0
    * KEGG.db: 3.2.3
    * annaffy: 1.50.0
    * crma: 2.50.0
    * olioClasses: 1.40.0
    * PFAM.db: 3.5.0
    * baySeq: 2.12.0
    * hu133plus2.db: 3.2.3
    * illuminaio: 0.20.0
    * bumphunter: 1.20.0
    * multtest: 2.34.0
    * sigenes: 1.52.0
    * DynDoc: 1.56.0
    * enoset: 1.34.0
    * NOISeq: 2.22.1
    * RNASeqPower: 1.18.0
    * interactiveDisplayBase: 1.16.0
    * AnnotationHub: 2.10.1
    * ProtGenerics: 1.10.0
    * AnnotationFilter: 1.2.0
    * ensembldb: 2.2.2
    * biovizBase: 1.26.0
    * OranismDbi: 1.20.0
    * gbio: 1.26.1
    * ReportinTools: 2.17.3
    * affycoretools: 1.50.6
    * TxDb.Hsapiens.UCSC.h19.knownGene: 3.2.2
    * Homo.sapiens: 1.3.1
    * BSenome.Hsapiens.UCSC.hg19: 1.4.0
    * AiMicroRna: 2.28.0
    * GenomeGraphs: 1.38.0
    * eneLenDataBase: 1.14.0
    * oseq: 1.30.0
    * KEGGraph: 1.38.0
    * KEGGprofile: 1.20.0
    * GEOquery: 2.46.15
    * RSpectra: 0.12-0
    * rARPACK: 0.11-0
    * mixOmics: 6.3.1
    * minfi: 1.24.0
    * FDb.InfiniumMethylation.h19: 2.2.0
    * methylumi: 2.24.1
    * lumi: 2.30.0
    * widetTools: 1.56.0
    * tkWidets: 1.56.0
    * Mfuzz: 2.38.0
    * venn: 1.5
    * maSiPro: 1.50.0
    * SPIA: 2.30.0
    * Gviz: 1.22.3
    * cummeRbund: 2.20.0
    * GenomicFiles: 1.14.0
    * derfinderHelper: 1.12.0
    * derfinder: 1.12.6
    * polyester: 1.14.1
    * Rsubread: 1.28.1
    * pcaMethods: 1.70.0
    * marray: 1.56.0
    * CGHbase: 1.38.0
    * siaR: 1.26.0
    * HCsnip: 1.17.0
    * metaenomeSeq: 1.20.1
    * dsfmt: 1.14.1
    * SNPRelate: 1.12.2
    * ALDEx2: 1.10.0
    * dada2: 1.6.0
    * CGEN: 3.14.0
    * chipseq: 1.28.0
    * DNAcopy: 1.52.0
    * EBImae: 4.20.1
    * fastse: 1.24.0
    * hopach: 2.38.0
    * seqLoo: 1.44.0
    * rGADEM: 2.26.0
    * MotIV: 1.34.0
    * PICS: 2.22.0
    * vsn: 3.46.0
    * Rino: 1.42.1
    * snpStats: 1.28.0
    * tilinArray: 1.56.0
    * IlluminaHumanMethylation450kmanifest: 0.4.0
    * quantsmooth: 1.44.0
    * GWASTools: 1.24.1
    * SeqArray: 1.18.2
    * SeqVarTools: 1.16.1
    * GENESIS: 2.8.1
    * Rhdf5lib: 1.0.0
    * ncdfFlow: 2.24.0
    * RProtoBufLib: 1.0.2
    * cytolib: 1.0.1
    * flowWorkspace: 3.26.9
    * TitanCNA: 1.16.0
    * HMMcopy: 1.20.0
    * flowStats: 3.36.0
    * mcmc: 0.9-5
    * MCMCpack: 1.4-2
    * clue: 0.3-54
    * flowClust: 3.16.0
    * openCyto: 1.16.1
    * rbenchmark: 1.0.0
    * pbs: 1.1
    * amm4: 0.2-5
    * rpreg: 3.1-2
    * RLRsim: 3.1-3
    * refund: 0.1-16
    * tmle: 1.3.0-1
